changes_feed_parameters.go

```diff
@@ -58,6 +59,10 @@ func (p ChangesFeedParams) String() string {
 	if len(p.SequenceNumber()) > 0 {
 		params = fmt.Sprintf("%v&since=%s", params, p.SequenceNumber())
 	}
-	logg.LogTo("TEST", "changesFeedParams: %v", params)
+	logg.LogTo("SYNCTUBE", "p.channels: %v", p.channels)
```

Comment:
I think we should either take this out or rename it to be a little more clear, like "Replication filter channels:"